xUnitTests/LoopringGraphTests/TestSearch.cs

```diff
@@ -52,13 +52,14 @@ public async void SearchL1Address(string address)
 
         [Theory]
         [InlineData("4baf35a6982a81402fbe5882a47a75add97a01cc69fc418b5fc545026751f08a")]
+        [InlineData("4BAF35A6982A81402FBE5882A47A75ADD97A01CC69FC418B5FC545026751F08A")]
         [InlineData("0x010a8144e644ace657426f10d35a2a3bee1d098e-0-0xdec45f94c65a79278b7a779a0f5e09a361672e5d-0x4baf35a6982a81402fbe5882a47a75add97a01cc69fc418b5fc545026751f08a-0")]
         public async void SearchNFT(string nftid)
         {
             var searchResult = await service.Search(nftid);
             Assert.NotEmpty(searchResult);
             Assert.IsType<NonFungibleToken>(searchResult![0]);
-            Assert.Contains(nftid, (searchResult![0] as NonFungibleToken)!.id);
+            Assert.Contains(nftid.ToLower(), (searchResult![0] as NonFungibleToken)!.id);
```

Comment:
Similar to above, `Assert.Contains(nftid, (searchResult![0] as NonFungibleToken)!.id, StringComparison.InvariantCultureIgnoreCase);` would be more explicit.